.github/workflows/release.yml

```
@@ -1,575 +1,562 @@
-        uses: rtCamp/action-slack-notify@v2
-        env:
+name: Flipper - CI/CD - Windows & Android & Google Play
+
+on:
+  # Enable manual run
+  workflow_dispatch:
+    inputs:
+      lane:
+        description: "Fastlane lane"
+        required: true
+        default: "internal"
```

actionlint (1.7.4)

default value "internal" of "lane" input is not included in its options "\"beta\", \"promote_to_production\", \"production\""

(events)